mongodbatlas/serverless_instances_test.go

```diff
@@ -359,6 +360,38 @@ func TestServerlessInstances_Update(t *testing.T) {
 	if diff := deep.Equal(serverlessInstance, expected); diff != nil {
 		t.Error(diff)
 	}
+
+	// Testing for removing tags
+	client, mux, teardown = setup()
```

Comment:
can you please group the tests using `t.Run` for example https://github.com/mongodb/go-client-mongodb-atlas/blob/774bf48ebd3cbee3dabcda9ac3100073ca2e2af2/mongodbatlas/organizations_test.go#L28-L173